assets/js/modules-admin.js

```diff
@@ -191,4 +191,33 @@ jQuery( document ).ready( function ( e ) {
             }
         });
     });
+
+    // Get Course modules (if any) on course select change
```

Comment:
The style in the code below is a little bit inconsistent. Maybe we could clean it up a bit?

- snake_case vs. camelCase variables
- spacing inside of parentheses

(Thinking in particular of the [JS coding style for Calypso](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md), but even just making it more consistent would be helpful I think)

Reply:
Good catch! I will change this up a bit :)